public/content/documentation/native/patterns/tidbit.md

```diff
@@ -0,0 +1,271 @@
+   - For enabled: Set `isEnabled` to `true`.
+   - For disabled: Set `isEnabled` to `false`. Announcement for disabled is "Dimmed"
+        - If necessary, you may change the accessibility trait of the _component to `notEnabled`, but this may overwrite the current accessibility role of the _component
+
+#### SwiftUI
+
+   - For selected, use `accessibilityAddTraits(.isSelected)`.
+   - For disabled, use view modifier `disabled()`.
+
+### Focus
+
+   - Use the device's default focus functionality. 
+   - External keyboard tab order often follows the screen reader focus, but sometimes this functionality requires additional development to manage focus.
+
+#### UIKit
+
+   - If VoiceOver is not reaching a particular element, set the element's `isAccessibilityElement` to `true`
+        - **Note:** You may need to adjust the programmatic name, role, state, and/or value after doing this, as this action may overwrite previously configured accessibility.
+   - Use `accessibilityViewIsModal` to contain the screen reader focus inside the modal.
+   - To move screen reader focus to newly revealed content, use `UIAccessibility.post(notification:argument:)` that takes in `.screenChanged` and the newly revealed content as the parameter arguments.
+   - To NOT move focus, but dynamically announce new content: use `UIAccessibility.post(notification:argument:)` that takes in `.announcement` and the announcement text as the parameter arguments.
+   - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements. 
+
+<!-- 
+Enter information for iOS Focus using SwiftUI, update below with appropriate details
+--> 
+
+### SwiftUI
+
+   - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+        - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+        - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+   - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcement examples
+
+   - "Important" (Exclamation point image) (grouped with text)
+   - All text is announced
+   - "Learn more, button" (Learn more CTA)
+
+## Android Developer Notes
+
+   - Initial screen reader focus can be on the entire tidbit and announce everything with or without a Call To Action (CTA). Or, the text and CTA can have separate focus.
+   - Meaningful images should have an accessibility label
+
+### Name
+
+   - Name describes the purpose of the control
+   - Programmatic name matches the visible text label (if any)
+
+#### Android Views
```

Comment:
"Android Views", "Jetpack Compose", and etc. have been bullets under "Name", "Role", etc. in other components. 

Not a huge deal so it's up to you but just noting for consistency.